src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -298,20 +306,15 @@ def do_clone():
 
         logger.info('mozilla-central cloned')
 
-    def build_files(self):
-        with open(os.path.join(self.repo_dir, '.mozconfig'), 'w') as f:
-            f.write('mk_add_options MOZ_OBJDIR=@TOPSRCDIR@/obj-firefox\n')
-            f.write('ac_add_options --enable-debug\n')
-            f.write('ac_add_options --enable-artifact-builds\n')
-
-        retry(lambda: run_check(['gecko-env', './mach', 'build'], cwd=self.repo_dir))
-        retry(lambda: run_check(['gecko-env', './mach', 'build-backend', '-b', 'ChromeMap'], cwd=self.repo_dir))
+    def configure_repo(self):
+        # Run ./mach python --version to create the virtualenv.
+        run_check(['gecko-env', './mach', 'python', '--version'], cwd=self.repo_dir)
```

Comment:
This wouldn't be strictly needed, as the `./mach python` calls in `rewrite_jsvm_lcov` would take care of this, but since they are running in multiple threads, they are often unable to create the virtualenv (as more than one process is trying to create it at the same time).